.github/workflows/e2e-olm/e2e-tests-olm.yml

```
@@ -0,0 +1,79 @@
+name: e2e-tests-olm
+description: Runs E2E tests against the OLM bundle
+on:
+  pull_request:
+
+    branches:
+    - v1alpha1
+
+env:
+  KIND_VERSION: "0.15.0"
+  GO111MODULE: "on"
+  OPERATOR_IMG: "quay.io/sustainable_computing_io/kepler-operator"
+  KUBECONFIG: /tmp/.kube/config
+  BUNDLE_IMG: "quay.io/sustainable_computing_io/kepler-operator-bundle"
+  OPERATOR_SDK_VERSION: "v1.27.0"
+ 
+
+jobs:
+  e2e:
+    name: build-operator
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        cluster_provider: [kind, microshift]
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+
+      - name: Install Go
+        uses: actions/setup-go@main
+        with: 
+          go-version-file: go.mod
+
+
+      - name: Use kepler action for kind cluster build
+        uses: sustainable-computing-io/kepler-action@v0.0.1
+        with:
+          runningBranch: ${{matrix.cluster_provider}}
+          cluster_provider: ${{matrix.cluster_provider}}
+
+      - name: Install tools
+        run: |
+          chmod +x tests/run-e2e.sh
+          ./test/run-e2e.sh tools_install 
```

How about 
```suggestion
          ./test/run-e2e.sh install-tools 
```
Also, can't the script check for the tools required and auto install those?